svc-systems/plugin/client.go

```diff
@@ -147,10 +152,22 @@ func (m *multiTargetClient) Get(uri string, opts ...CallOption) response.RPC {
 			log.Printf("execution of GET %s on %s plugin returned non 2xx status code; %v", uri, target.ID, resp.Body)
 		}
 	}
-
+	// Checking whether the struct passed as the interface has a ReqURI field.
+	// If it is a collection request then the ReqURI field won't be present.
+	// Only struct associating with individual resource collection requires the ReqURI field.
+	field := reflect.ValueOf(m.call.collector).Elem().FieldByName("ReqURI")
+	if field.IsValid() {
+		chassisID := getChassisID(uri)
+		field.SetString(chassisID)
+	}
 	return m.call.collector.GetResult()
 }
 
+func getChassisID(uri string) string {
+	parts := strings.Split(uri, "/")
+	return parts[len(parts)-1]
+}
+
 func (m *multiTargetClient) Post(uri string, body *json.RawMessage) response.RPC {
 	panic("implement me")
```

Comment:
panic exits ? we cannot afford to exit. Even if this is restarted it will panic again. so return a method not implemented error

Reply:
I will fix these in the upcoming PRs

Reply:
I has asked for the task no. to be added here

Reply:
@Bharath-KKB the task created for this is task check list 5 of #390

Reply:
@Bharath-KKB the jira subtask for this is https://jira-pro.its.hpecorp.net:8443/browse/BRUCE-4208